libs/platform/src/lib/components/upload-collection/upload-collection/upload-collection.component.html

```diff
@@ -177,16 +177,26 @@
 
                         <ng-template #editMode>
                             <div fd-form-item>
-                                <input
-                                    fd-form-control
-                                    [value]="_getItemName(item)"
-                                    (blur)="_fileNameChange($event, item)"
-                                    type="text"
-                                    i18n-placeholder
-                                    placeholder="Enter a name"
-                                    compact="true"
-                                />
-                                <span [innerText]="_getItemExtension(item)"></span>
+                                <fd-form-input-message-group [triggers]="[]" [isOpen]="!!item.sameFilenameState">
```

Comment:
Do we need to pass empty array?

Reply:
yes, because by default `triggers` has predefined value but it not need in this case
```typescript
    @Input()
    triggers: string[] = ['focusin', 'focusout'];
```